modules/katamari/src/test/ts/atomic/api/struct/AdtTest.ts

```diff
@@ -51,13 +51,14 @@ UnitTest.test('ADT Test', () => {
     { actual: [ 'target', 'block' ] }
   ]);
 
-  const none = function () {
-    Assert.eq('eq', 0, arguments.length);
+  const none = (...args: any[]) => {
+    Assert.eq('eq', 0, args.length);
   };
 
   const targetStr = 'target';
   const blockStr = 'block';
 
+  // eslint-disable-next-line prefer-arrow/prefer-arrow-functions
   const tag = function (target, block) {
     Assert.eq('eq', 2, arguments.length);
```

Comment:
```suggestion
  const tag = (target, block, ...args) => {
    Assert.eq('eq', 0, arguments.length);
```

Reply:
This won't work sorry, as you can't use `arguments` with arrow functions.